iso8859-1/src/lib.rs

```diff
@@ -139,7 +139,7 @@ impl IsoLatin1Char {
     /// TODO
     /// ```
     pub fn is_numeric(&self) -> bool {
-        todo!()
+        self >= &IsoLatin1Char(0x30) && self <= &IsoLatin1Char(0x39)
```

Comment:
There is a better syntax for this. You can get the anonymous field using `self.0`, e.g. `self.0 >= 0x30`